### PR TITLE
Desc english

### PR DESCRIPTION
**Pull request by youngant**
This should solve #15. Adds a column indicating if the description is in English. The recognition is slow (takes a few minutes), so this should really only be run once.

### CODE REVIEW

**Comment by youngant** [comment]
Do I just need to add the missing package to the setup.py to fix the failed check? No way to fix that automatically?

**Comment by adam-broussard** [comment]
Hmm I'm not sure... You'll definitely want to add it to `setup.py` because if we import it, it is a dependency.  That may or may not fix the linter - if not, you'll likely have to add it to the [pip install](https://github.com/adam-broussard/bestreads/blob/b5c6d5bbfb94dbfd8cc27d4fac0d19788ef506a3/.github/workflows/pylint.yml#L24) for the environment the linter is run in.  I would love for that to just pull the dependencies from `setup.py`, but I'm not sure how to make it happen...  It might be as simple as getting the environment to install `bestreads`, in which case it should also install the relevant listed dependencies so we only have to change it in one place?

**Comment by adam-broussard** [comment]
[This](https://github.com/astropy/astropy/blob/main/pip-requirements) seems to indicate that my comment above is correct - maybe you could try getting rid of the packages I added in the yml file and have it just install bestreads itself (with langdetected added to setup.py)?  I think that should do it.

**Comment by youngant** [comment]
I think the package installation is fixed now. All we should need to do is add the package to setup.py.

File: bestreads/text.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    except LangDetectException:
+        return False
+
+def add_english_column(data):
+    """
+    Adds a new column to a DataFrame indicating if the 'description' column is
+    in English.
+
+    Args:
+        data (pandas.DataFrame): The DataFrame constaining the descriptions of the books.
```
Comment (adam-broussard) [inline bestreads/text.py:30]:
```suggestion
        data (pandas.DataFrame): The DataFrame containing the descriptions of the books.
```

**Review by adam-broussard — CHANGES_REQUESTED** [review]
Check these out and let me know what you think!

File: bestreads/text.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        english_bool (bool): Indicates if the text is English.
+    """
+    if not isinstance(text, str):
+        return False
+    try:
+        return detect(text) == 'en'
+    except LangDetectException:
+        return False
+
+def add_english_column(data):
```
[the remark this answers is not in the document]
Comment (youngant) [reply bestreads/text.py:24]: It's probably better to return a copy like you suggested.

**Comment by adam-broussard** [comment]
Just as a heads-up, there is a re-request review button where you select the reviewer, so if you are done with changes, you can click that to ping me to start the approval process again!

**Comment by youngant** [comment]
Thanks, I wasn't sure if there was a way to do something like that.

**Review by adam-broussard — APPROVED** [review]

**Pull request merged**